README.md

```diff
@@ -234,6 +234,33 @@ async create(@requestBody() role: Role): Promise<Role> {
 }
 ```
 
+- Or we could use the OpenAPI operation based decorators -
+  - `authorizedGet`
+  - `authorizedPost`
+  - `authorizedPut`
+  - `authorizedPatch`
+  - `authorizedDel`
+
+  these decorators are an extended version of the loopback operation decorators - `get`, `post`, `put`, `patch` and `del` respectively, with an extra parameter for authorization metadata. These decorators have an added benefit of specifying the required permissions in the OpenAPI description of the endpoints.
+
+```ts
+authorizedPost(rolesPath, {
```

Comment:
can this be an additional decorator rather than overriding the existing one ?

Reply:
done, added a new decorator - authorizeWithSpecs